crates/ruff_python_parser/src/error.rs

```diff
@@ -84,6 +84,42 @@ impl std::fmt::Display for FStringErrorType {
     }
 }
 
+/// Represents the different types of errors that can occur during parsing of a t-string.
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum TStringErrorType {
```

Comment:
Same as elsewhere. I don't think we have to duplicate the enum here and instead can use it for both f and t-string (the outer variant is enough to distinguish the two cases and even then, it's not clear to me if that's even required)